README.md

```diff
@@ -1,39 +1,41 @@
-
-## Process to run the pipelines
-
-The pipeline requires the Nextflow Tower for the application level monitoring. A free test account can be created for evaluation purposes at https://tower.nf/.
+1. Have access to a Google Cloud account
+2. Have a Nextflow tower account
+3. Have a list of samples in a .tsv file e.g:
+	https://github.com/enasequence/covid-sequence-analysis-workflow/blob/demo/TSVs/illumina.tsv
 
 ### Preparation
 
-1. Store `export TOWER_ACCESS_TOKEN='...'` in `$HOME/.bash_profile`. Restart the current session or source the updated `$HOME/.bash_profile`.
-2. Run `git clone https://github.com/enasequence/covid-sequence-analysis-workflow`.
-3. Create `./covid-sequence-analysis-workflow/data/projects_accounts.csv` with submission_account_id and submission_passwor, for example:
->  project_id,center_name,meta_key,submission_account_id,submission_password,ftp_password
->  PRJEB45555,"European Bioinformatics Institute",public,,,
-
-### Running pipelines
-
-1. Run `./covid-sequence-analysis-workflow/init.sra_index.sh` to initialize or reinitialize the metadata in BigQuery.
-2. Run `./covid-sequence-analysis-workflow/./start.lsf.jobs.sh` with proper parameters to start the batch jobs on LSF or `./covid-sequence-analysis-workflow/./start.gls.jobs.sh` with proper parameters to start the batch jobs on GCP.
-
-### Error handling
-
-If a job is killed or died, run the following to update the metadata to avoid reprocessing samples completed successfully.
-
-1. Run `./covid-sequence-analysis-workflow/update.receipt.sh <batch_id>` to collect the submission receipts and to update submission metadata. The script can be run at anytime. It needs to be run if a batch job is killed instead of completed for any reason.
-2. Run `./covid-sequence-analysis-workflow/set.archived.sh` to update stats for analyses submitted. The script can be run at anytime. It needs to be run at least once before ending a snapshot to make sure that the stats are up-to-date.
-
-To reprocess the samples failed, delete the record in `sra_processing`.
+1. Follow the setup instructions as described on Nextflow tower to setup a [compute environment](https://help.tower.nf/22.4/compute-envs/google-cloud-lifesciences/) on Google
+2. Follow the setup instructions as described on Nextflow tower to integrate the [codebase](https://help.tower.nf/22.4/git/overview/#github)
+3. Create a project on GCP and a [Bucket](https://console.cloud.google.com/storage/browser) with the following files and folders:
+	a. Folders - data,results,scratch and storeDir
+	b. Files - illumina5.tsv, nanopore5.tsv, data/projects_accounts.csv, [data/NC_045512.2.fa.fai](https://github.com/enasequence/covid-sequence-analysis-workflow/blob/5b6b53b3a228ec55cb1b34bbc3aec5ec31759edd/illumina/illumina.nf#L12), [data/NC_045512.2.fa](https://github.com/enasequence/covid-sequence-analysis-workflow/blob/5b6b53b3a228ec55cb1b34bbc3aec5ec31759edd/illumina/illumina.nf#L11)
```

Comment:
1. It is good that you did not include a copy of data/projects_accounts.csv. Perhaps, it is worth describing what it is about.
2. Would readers expecting to see links to data/NC_045512.2.fa.fai and data/NC_045512.2.fa files themselves or where they are referenced?